terraform/modules/api-gateway/lambda.tf

```diff
@@ -5,10 +5,16 @@ resource "aws_lambda_function" "api_gateway_lambda" {
   handler       = "api_gateway_lambda.handler"
```

Comment:
What was your thinking behind not using the terraform module that we've used for other lambdas? Like the ingestion lambda for example. A lot of the IAM attachment and overhead would have been abstracted away

Reply:
I’ve replaced the manual Lambda definition with the Terraform Lambda module, following the same structure as the ingestion Lambda